lib/favorites/index.js

```diff
@@ -25,8 +25,9 @@ exports.create = options => {
 
   var buildFavoriteAddition = function(type) {
     return (postOptions, callback) => {
-      postOptions.type = type;
-      return handleFavorites(postOptions, callback);
+      var options = JSON.parse(JSON.stringify(postOptions));
```

Comment:
Should we use JSON parse -> stringify, or can we use a deep clone of some sort w/ Underscore?

Reply:
I couldn't find an underscore deep clone. If there is one, I'd be happy to use that instead though